canonical/charmhub.io
# Add packages page to beta store

## Done
Setup the React app for the beta store page and fetching packages

## How to QA
- Go to https://charmhub-io-1555.demos.haus/beta-store
- Check that cards are loaded on the page

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-2587
